src/component/administration/customization/CustomAttributeEdit.tsx

```diff
@@ -270,6 +388,50 @@ export const CustomAttributeEdit: React.FC = () => {
                 />
               </Col>
             </Row>
+            {domain === VocabularyUtils.RDF_STATEMENT && (
```

Comment:
I suggest extracting the selector and its related code into a separate component, it basically doubled the amount of code in this file.

I also think it would make sense to allow adding new annotated relationships when editing an existing custom attribute, so it should not be disabled (just check when the user is editing that they are not removing any previously selected values).